### ISSUE TITLE
Include JLPT tag in API word search result

### ISSUE THREAD

**Issue by Nyankoo** [WeDontPanic/Jotoba#35]
**Is your feature request related to a problem? Please describe.**
When using the API, found words don't include their JLPT tag while they're displayed on the website search

**Describe the solution you'd like**
The "words" json entry should contain the JLPT tag of that word 

**Describe alternatives you've considered**

**Additional context**

**Comment by JojiiOfficial** [comment]
I agree. I've already encountered this issue but unfortunately I don't want to change the API since it could break existing programs. I'm going to add a new version of the API which will include the JLPT information.

**Comment by Nyankoo** [comment]
Thank you for the quick reply and I'm looking forward to it!

**Comment by Nyankoo** [comment]
@JojiiOfficial Just quickly checking in on the progress of this.

**Comment by JojiiOfficial** [comment]
This is currently out of scope. There is a new, undocumented, API which you could use but this can be changed later at any time. For a V2 I'd like to wait and collect more data that will be included in the new API version so we don't have to change the API all the time.

**Comment by keithfancher** [comment]
I came here to suggest the very same thing :D

I also wanted to suggest including the "main" example sentence that appears when you search for a term in the API search results.

Either that OR tag/sort the results returned when calling `/api/search/sentences`, so we can grab the "top" sentence. Right now, the sentences on the main search results aren't even guaranteed to be in that list, I realized. (Try searching via the API for `嵐`, for example.)

 I'd be happy to file a separate issue for this request if you'd prefer!

Separately, I also wanted to say: thank you for jotoba! It's awesome.

**Comment by keithfancher** [comment]
Oh, I just found the [trello board](https://trello.com/c/lNU0g3M9/464-api-v2-ideas)... I guess we're on the same page :smile: 

![Screenshot from 2024-03-14 14-36-38](https://github.com/WeDontPanic/Jotoba/assets/132365/8b183cf6-d18d-4b8e-a3ef-7eefd07908a6)
